lib/kitchen/directions/bake_example.rb

```diff
@@ -22,7 +22,8 @@ def self.v1(example:, number:, title_tag:, numbered_solutions: false)
 
         example.titles.each do |title|
           if title.parent.has_class?('os-caption-container') || \
-             title.parent.has_class?('os-caption')
+             title.parent.has_class?('os-caption') || \
+             title.parent.name == 'caption'
```

Comment:
miiiight be good to enforce this with a spec in case someone comes across it later and tries to change it -- i think the `BakeExample` spec has something in there already [about tables](https://github.com/openstax/kitchen/blob/main/spec/directions/bake_example_spec.rb#L136), so this would just be a matter of adding a different table input format

Reply:
done and done